Formula/mingw-w64.rb

```diff
@@ -148,6 +148,18 @@ def install
         system "make", "install"
       end
 
+      args = %W[
+        --host=#{target}
+        --with-sysroot=#{arch_dir}/#{target}
+        --prefix=#{arch_dir}
+        --program-prefix=#{target}-
+      ]
+      mkdir "mingw-w64-tools/widl/build-#{arch}" do
+        system "../configure", *args
+        system "make"
+        system "make", "install"
+      end
```

Comment:
What does this build?

Reply:
widl is a MIDL compiler.

```
An MIDL compiler processes the IDL file and creates a type library, a header, and proxy files.
```

Part of reason users really wanted a `wine` formula was to have this utility, mingw luckily now wine maintainers also work on mingw and keeping there headers/widl in sync with upstream wine.

Reply:
Hm, ok. Can you double-check how much this adds to the installed bottle size?

Reply:
widl is a lone binary each sized around 610KB, not sure about writing a test for this due to lack of familiarity with writing an idl file.

Reply:
Not even a minimal `"hello world"`-type test? It seems weird to try to enable functionality in a formula when we don't even know how to use it.

Reply:
I've added a basic test that compiles `example.idl` into `example_s.c`, how would I test this file is created? (guessing I need to verify it was created)

Reply:
Something like
```ruby
assert_predicate testpath/"example_s.c", :exist, "example_s.c should have been created"
```
should work.

Reply:
Done, rebased and pushed